packages/datepicker/datepicker.scss

```diff
@@ -0,0 +1,70 @@
+@import "~@fremtind/jkl-core/build/scss/variables/spacing.scss";
+@import "~@fremtind/jkl-core/build/scss/variables/colors.scss";
+@import "~@fremtind/jkl-core/build/scss/typography/paragraphs.scss";
+@import "~@fremtind/jkl-core/build/scss/functions.scss";
+@import "~@fremtind/jkl-text-input/text-input.scss";
```

Comment:
We should avoid importing other stylesheets directly, as this will duplicate the rules (presuming `text-input` is used elsewhere).